src/main/java/com/onarandombox/MultiverseCore/enums/KeepWorld.java

```diff
@@ -0,0 +1,19 @@
+package com.onarandombox.MultiverseCore.enums;
+
+/**
+ * Custom enum for method argument in deciding what to keep when deleting world files.
+ */
+public enum KeepWorld {
```

Comment:
hmm, maybe it is better to have the enum for "what to delete", as compared to "what to keep". Perhaps a WorldDeleteMode enum with ALL, WORLD_AND_CONFIG, WORLD.